engine/shared/map/init.lua

```diff
@@ -361,17 +361,6 @@ end
 
 accessor( map, "properties" )
 
-function map:getTileset( layer )
```

Comment:
Should this be exposed as `map:getTilesets()` or some equivalent in Grid Engine 10? The next major version will include these changes.

Reply:
Yes, I think that's a good idea.